{"message":"Transport already attached: file, assign a different name","stack":"Error: Transport already attached: file, assign a different name\n    at exports.Logger.Logger.add (C:\\Users\\pfit\\Desktop\\Node\\node_modules\\winston\\lib\\winston\\logger.js:487:11)\n    at Object.winston.<computed> [as add] (C:\\Users\\pfit\\Desktop\\Node\\node_modules\\winston\\lib\\winston.js:86:34)\n    at module.exports (C:\\Users\\pfit\\Desktop\\Node\\startup\\loggingErrors.js:29:9)\n    at Object.<anonymous> (C:\\Users\\pfit\\Desktop\\Node\\index.js:12:35)\n    at Module._compile (internal/modules/cjs/loader.js:1063:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)\n    at Module.load (internal/modules/cjs/loader.js:928:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:769:14)\n    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)\n    at internal/main/run_main_module.js:17:47","level":"error","timestamp":"2021-03-07T15:45:22.707Z"}
{"message":"Transport already attached: file, assign a different name","stack":"Error: Transport already attached: file, assign a different name\n    at exports.Logger.Logger.add (C:\\Users\\pfit\\Desktop\\Node\\node_modules\\winston\\lib\\winston\\logger.js:487:11)\n    at Object.winston.<computed> [as add] (C:\\Users\\pfit\\Desktop\\Node\\node_modules\\winston\\lib\\winston.js:86:34)\n    at module.exports (C:\\Users\\pfit\\Desktop\\Node\\startup\\loggingErrors.js:29:9)\n    at Object.<anonymous> (C:\\Users\\pfit\\Desktop\\Node\\index.js:12:35)\n    at Module._compile (internal/modules/cjs/loader.js:1063:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)\n    at Module.load (internal/modules/cjs/loader.js:928:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:769:14)\n    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)\n    at internal/main/run_main_module.js:17:47","level":"error","timestamp":"2021-03-07T15:45:22.780Z"}
{"message":"Transport already attached: file, assign a different name","stack":"Error: Transport already attached: file, assign a different name\n    at exports.Logger.Logger.add (C:\\Users\\pfit\\Desktop\\Node\\node_modules\\winston\\lib\\winston\\logger.js:487:11)\n    at Object.winston.<computed> [as add] (C:\\Users\\pfit\\Desktop\\Node\\node_modules\\winston\\lib\\winston.js:86:34)\n    at module.exports (C:\\Users\\pfit\\Desktop\\Node\\startup\\loggingErrors.js:29:9)\n    at Object.<anonymous> (C:\\Users\\pfit\\Desktop\\Node\\index.js:12:35)\n    at Module._compile (internal/modules/cjs/loader.js:1063:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)\n    at Module.load (internal/modules/cjs/loader.js:928:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:769:14)\n    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)\n    at internal/main/run_main_module.js:17:47","level":"error","timestamp":"2021-03-12T13:24:51.794Z"}
{"level":"info","message":"Listening on port 3000...","timestamp":"2021-03-12T13:28:16.609Z"}
{"message":"failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1146:16) {\n  name: 'MongoNetworkError'\n}]","name":"MongoNetworkError","stack":"MongoNetworkError: failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1146:16) {\n  name: 'MongoNetworkError'\n}]\n    at Pool.<anonymous> (C:\\Users\\pfit\\Desktop\\Node\\node_modules\\mongodb-core\\lib\\topologies\\server.js:503:11)\n    at Pool.emit (events.js:315:20)\n    at Connection.<anonymous> (C:\\Users\\pfit\\Desktop\\Node\\node_modules\\mongodb-core\\lib\\connection\\pool.js:326:12)\n    at Object.onceWrapper (events.js:422:26)\n    at Connection.emit (events.js:315:20)\n    at Socket.<anonymous> (C:\\Users\\pfit\\Desktop\\Node\\node_modules\\mongodb-core\\lib\\connection\\connection.js:245:50)\n    at Object.onceWrapper (events.js:422:26)\n    at Socket.emit (events.js:315:20)\n    at emitErrorNT (internal/streams/destroy.js:106:8)\n    at emitErrorCloseNT (internal/streams/destroy.js:74:3)\n    at processTicksAndRejections (internal/process/task_queues.js:80:21)","level":"error","timestamp":"2021-03-12T13:28:16.631Z"}
